inc/Ui/Admin.php

```diff
@@ -75,16 +76,16 @@ protected function showVersion() {
      * it verifies either:
      *   'savedir' has been moved elsewhere, or
      *   has protection to prevent the webserver serving files from it
+     *
+     * The actual check is carried out via JavaScript. See behaviour.js
      */
     protected function showSecurityCheck() {
         global $conf;
         if(substr($conf['savedir'], 0, 2) !== './') return;
         $img = DOKU_URL . $conf['savedir'] .
             '/dont-panic-if-you-see-this-in-your-logs-it-means-your-directory-permissions-are-correct.png';
-        echo '<a style="border:none; float:right;"
-                href="http://www.dokuwiki.org/security#web_access_security">
-                <img src="' . $img . '" alt="Your data directory seems to be protected properly."
-                onerror="this.parentNode.style.display=\'none\'" /></a>';
+        echo '<a style="border:none; float:right;" id="security__check"
+                href="http://www.dokuwiki.org/security#web_access_security" data-src="' . $img . '">⚠</a>';
```

Comment:
Do we actually need this `⚠` sign? Also should we make the link to https?

Reply:
The `⚠` is there to not have an empty link when for some reason the javascript is not executed. We could place some explanatory text there, but that would probably needed to be translated. I believe noJS will happen very rarely so I thought a simple icon is the more international approach and good enough. It also makes an okay search term as in "why is there an exclamation mark icon on my admin page?".

I will fix the URL to https.